### ISSUE TITLE
QUESTION: unable to spawn printcontenv

### ISSUE THREAD

**Issue by PriamX** [just-containers/s6-overlay#362]
Hello.

I'm not even sure if this is an s6-overlay issue, Google brought me here. 

After doing a Fedora update (and rebooting) 5 out of 15 of my docker containers provided by linuxserver.io are getting the following error and failing to boot up:

`backtick: fatal: unable to spawn printcontenv: No such file or directory`

Deleting the container and image, and redeploying makes no difference. Even tried a previous version of the container. 

The folks on linuxserver.io's Discord were at a loss, so I thought I'd ask here. Could this be related to s6-overlay, and if so, where can I start to troubleshoot?

Thanks!

**Comment by skarnet** [comment]
This sounds like an usrmerge problem with `/bin/execlineb` (which is used to interpret the `printcontenv` script) becoming unavailable. @jprjr, can you confirm? and what is an appropriate solution?

(The next version of s6-overlay, currently in testing, will solve all this, but it's not ready yet.)

**Comment by jprjr** [comment]
Can you provide an example dockerfile, and some steps to reproduce?

**Comment by PriamX** [comment]
Yes, here's one of the five, Ombi.

```
version: "2"
networks:
  plexnet:
    external:
      name: plexnet
services:
 ombi:
    image: linuxserver/ombi:latest
    container_name: ombi
    environment:
      - PUID=981
      - PGID=100
      - TZ=America/Denver
      - BASE_URL=/ombi #optional, for reverse proxy
    volumes:
      - /media/docker/appdata/ombi:/config
    dns:
      - 8.8.8.8
      - 8.8.4.4
    networks:
      - plexnet
    depends_on:
      - swag
    restart: unless-stopped
```

**Comment by PriamX** [comment]
The only steps to reproduce is to deploy this container, the container never boots up and continues to restart, each time with the error in the log.

**Comment by skarnet** [comment]
OK this is not a `/bin/execlineb` problem else the container would never reach that point. It sounds like something is simply wrong with `printcontenv`. Is there a `/usr/bin/printcontenv` file in the container, once s6-overlay has been applied?

**Comment by PriamX** [comment]
Well, I'm not having a lot of luck trying to figure out if `printcontenv` is in there. The container won't start so I can't shell in. So I tried a `docker save` instead and that didn't work either, it threw the below error and the resulting tar file appears incomplete or corrupted:

```
[root@mediaserv docker-extract]# docker save linuxserver/ombi > ombi.tar
Error response from daemon: open /media/docker/root/overlay2/9e08fd0a1828ecab9f3dec0600036758b5a06bfa7f47a15019f063ad60b82b19/diff/etc/selinux/semanage.conf: no such file or directory
[root@mediaserv docker-extract]# ls
ombi.tar
[root@mediaserv docker-extract]# tar vxf ombi.tar
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors
[root@mediaserv docker-extract]#
```

**Comment by PriamX** [comment]
Oh, and just prior to that I deleted the image and re-pulled it from docker hub (using the compose file from my previous post).

**Comment by PriamX** [comment]
Well...I  fixed it. Unfortunately, I don't know how I fixed it, or why the problem occurred in the first place. Out of desperation, I removed all my containers (working or not), deleted all the container images, shut down docker and containerd, removed a the rpms for docker-ce* and containerd. rebooted, did a `dnf update`, rebooted, re-installed docker-ce and containerd, then re-deployed my containers from my compose files. And they're working now.
